WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -505,4 +506,12 @@ public void onEventMainThread(NotificationEvents.NoteModerationFailed event) {
 
         EventBus.getDefault().removeStickyEvent(NotificationEvents.NoteModerationFailed.class);
     }
+
+    public SiteModel getSelectedSite() {
+        if (getActivity() instanceof WPMainActivity) {
+            WPMainActivity mainActivity = (WPMainActivity) getActivity();
+            mainActivity.getSelectedSite();
```

Comment:
Good eyes, I actually forgot to replace this `getSelectedSite` like I did in other places (https://github.com/wordpress-mobile/WordPress-Android/pull/4455/commits/6884f6b19836565cab927100fbbb232e5c8e2142). I'll pass the site to that fragment instead of having this method.